src/script/calling/Call.ts

```diff
@@ -81,6 +90,35 @@ export class Call {
     return this.participants().find(({user, clientId}) => user.isMe && this.selfClientId === clientId);
   }
 
+  addAudio(audioId: string, stream: MediaStream) {
+    this.audios[audioId] = {audioElement: null, stream};
+  }
+
+  removeAudio(audioId: string) {
+    delete this.audios[audioId];
```

Comment:
How about stopping the stream before removing it?

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10657/commits/487f5113d8bed815d1c9a5a51f749303f5ebf985